test/price-import.spec.coffee

```diff
@@ -142,6 +143,44 @@ describe 'PriceImport', ->
       expect(actual).toEqual(expected)
       done()
 
+  describe '_removeEmptyPriceFields', ->
+  
+    it 'should remove empty object fields', (done) ->
+      pricesWithEmptyFields = [
+        mockPrice({customerGroup: { typeId: "customer-group",id: "" } }),
+        mockPrice({country: 'SE', currency: "SEK", amount: 2000}),
+        mockPrice({currency: "SEK", amount: 2000}),
+      ]
+      pricesWithEmptyFields[0].country = ''
+      pricesWithEmptyFields[0].value.centAmount = '' ## make sure we do not remove when centAmount is empty
+      pricesWithEmptyFields[2].country = ''
+      
+      _removeEmptyPriceFields = @import._removeEmptyPriceFields
+      cleanedUpPrices = pricesWithEmptyFields.map (price) -> _removeEmptyPriceFields(price)
+
+      expect(cleanedUpPrices[0]).toEqual(jasmine.objectContaining({
+        value: { currencyCode: 'EUR', centAmount: '' }
+      }))
+      expect(Object.keys(cleanedUpPrices[0])).toNotContain('country')
+      expect(Object.keys(cleanedUpPrices[0])).toNotContain('customerGroup')
+
+      expect(cleanedUpPrices[1]).toEqual(jasmine.objectContaining({
+        value: { currencyCode: 'SEK', centAmount: 2000 },
+        country: 'SE',
+        customerGroup: jasmine.objectContaining({ typeId: 'customer-group' })
```

Comment:
I am not sure why you expect `customer-group` if second price mock does not have it set?

Reply:
It's set here: https://github.com/sphereio/sphere-product-import/blob/d70f27521817455224ab3354244b442c42d3ec2f/test/price-import.spec.coffee#L15

First mock set's it together with an empty string `id` and should therefore be removed. But this one sets a correct id with the help of `cuid()`